.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,26 @@
+# Copied from GitHub docs:
+# https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request
+
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
do we have to create this token? currently for this repo we don't have secrets for [dependabot](https://github.com/aws/aws-encryption-sdk-python/settings/secrets/dependabot)
only for [actions](https://github.com/aws/aws-encryption-sdk-python/settings/secrets/actions)

Reply:
Nope, the token is [automatically provided to Actions workflows](https://docs.github.com/en/actions/security-guides/automatic-token-authentication). Dependabot workflows get [read-only tokens by default](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions#changing-github_token-permissions), but the `permissions` block in the workflow config serve to allow the necessary actions for merging.